src/scripts.js

```diff
@@ -22,20 +25,64 @@ window.addEventListener('load', fetchData())
 function fetchData() {
     Promise.all([getData('customers'), getData('rooms'), getData('bookings')])
     .then(data => {
-        console.log(data)
         allCustomers = data[0].customers
         allRooms = data[1].rooms
         allBookings = data[2].bookings
-        // displayCustomerData(user)
+        currentCustomer = new Customer(allCustomers[Math.floor(Math.random() * allCustomers.length)])
+        displayCustomerData(currentCustomer, allRooms, allBookings)
     })
 }
 
-// function displayCustomerData(user) {
-//     //Hard code user until we have log in page?
-//     //Add user name in header
-//     //Add total cost in header
-//     //Display all past bookings as cards
-//     //Display all upcoming bookings as cards
-// }
+function displayCustomerData(currentCustomer, allRooms, allBookings) {
+    greeting.innerText = `Welcome, ${currentCustomer.name}!`
+    cost.innerHTML = `Your total cost of bookings is $${currentCustomer.getTotalCost(allBookings, allRooms).toFixed(2)}.`
+    let userAllBookings = currentCustomer.getAllBookings(allBookings)
```

Comment:
I would put this in your Customer class, so the Customer class holds the list of bookings. This would keep your data model together.